### DISCUSSION TITLE
p-dialog and calendar issue (calendar is cropping in dialog box)

### DISCUSSION THREAD

**Discussion by sim-niron** [primefaces/community#737] [PrimeNG]
In dialog box my calendar-input is cropping out
see image:

![prime-issue](https://github.com/primefaces/community/assets/120708183/181e72dc-1fb7-493a-a748-79d1fb51c026)

**Comment by SoyDiego** [comment]
> In dialog box my calendar-input is cropping out see image:
> 
> ![prime-issue](https://private-user-images.githubusercontent.com/120708183/291887128-181e72dc-1fb7-493a-a748-79d1fb51c026.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMzNTQ0OTksIm5iZiI6MTcwMzM1NDE5OSwicGF0aCI6Ii8xMjA3MDgxODMvMjkxODg3MTI4LTE4MWU3MmRjLTFmYjctNDkzYS1hNzQ4LTc5ZDFmYjUxYzAyNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIyM1QxNzU2MzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMmI5YzE5OTIwMzZkZjAxMzMzMjY3YWE4ZjQ2OGQwNTMyMTZjYWY2ZWQzZGJiYjljMGQyYjMzMTAzZGY2YzdlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.9svelkXNsnreVKNOSHEvTMrYORlNv4wFMtBxKg2DvxA)

Please generate a Stackblitz and maybe we can help

**Comment by cetincakiroglu** [comment]
Closing the topic due to the inactivity.
